### ISSUE TITLE
Committment discounts show blank

### ISSUE THREAD

**Issue by saleemmy** [microsoft/finops-toolkit#867]
I have lot of RI purchased but the commitments/RI  savings show blank in both the cost summary and rate optimization reports.

**Comment by flanakin** [comment]
@saleemmy By any chance, are you using FOCUS 1.0? 

There was a bug that was fixed recently in Cost Management exports where PricingCategory was set to "Standard" for commitment discount usage. FTK reports filter on PricingCategory == "Committed", so this is why the bug ends up showing no commitment discounts. The bug was fixed in late July, so any exports that ran at the end of the month should have fixed this however. If you're seeing this, rerun the exports and you should be good. We'll also add a local workaround for this in 0.5.

**Comment by saleemmy** [comment]
@flanakin 
I just reinstalled finops hub toolkit from the site, and I am sure it is 0.4 version. The exports are Focus 1.0 (preview). I have manually exported all the previous months for both daily and monthly settings. 
I am supposed to see more than 100K of RI and SP savings but see only 4K which is for the SP savings and all RI savings for both VM and OS show zero. Even storage RI shows zero savings.
The SLES OS RI show as Virtual Machines service instead of SUSE Linux just like in the Azure portal.
